policyengine_us/parameters/gov/states/ga/tax/income/exemptions/personal.yaml

```
@@ -0,0 +1,22 @@
+description: Georgia phases-out the exemptions out at this amount depending on the filing status.
+metadata:
+  period: year
+  unit: currency-USD
+  label: Georgia dependent phase out amount
+  
+  reference:
+  - title: 2022 Form 500 Georgia Income Tax Return
+    href: https://apps.dor.ga.gov/FillableForms/PDFViewer/Index?form=2022GA500
+  - title: Official Code of Georgia Annotated | CH17 Income Taxes | O.C.G.A. § 48-7-26
+    href: https://advance.lexis.com/documentpage/?pdmfid=1000516&crid=2c053fd5-32c1-4cc1-86b0-36aaade9da5b&pdistocdocslideraccess=true&config=00JAA1MDBlYzczZi1lYjFlLTQxMTgtYWE3OS02YTgyOGM2NWJlMDYKAFBvZENhdGFsb2feed0oM9qoQOMCSJFX5qkd&pddocfullpath=%2Fshared%2Fdocument%2Fstatutes-legislation%2Furn%3AcontentItem%3A6348-G0H1-DYB7-W3JT-00008-00&pdcomponentid=234187&pdtocnodeidentifier=ABWAALAADAAL&ecomp=k2vckkk&prid=4862391c-e031-443f-ad52-ae86c6bb5ce2
+
+JOINT:
+  1986-01-01: 3_700
```

To match the law and avoid needing to multiply in the formula
```suggestion
  1986-01-01: 7_400
```